Authentication/SCRAM-SHA-1/XMPPSCRAMSHA1Authentication.m

```diff
@@ -65,9 +65,21 @@ + (NSString *)mechanismName
 
 - (id)initWithStream:(XMPPStream *)stream password:(NSString *)password
 {
+	return [self initWithStream:stream username:nil password:inPassword];
+}
+
+- (id)initWithStream:(XMPPStream *)stream username:(NSString *)inUsername password:(NSString *)inPassword
```

Comment:
Did you mean to use `password` instead of `inPassword` for the last param? The compiler will complain at line 83